android-DL4JIrisClassifierDemo.md

```diff
@@ -0,0 +1,264 @@
+---
+title: Using Deeplearning4J in Android Applications - demo 1
+layout: default
+---
+# Using Deeplearning4J in Android Applications 
+### DL4JIrisClassifierDemo
+The example application trains a small neural network on the device using Anderson’s Iris data set for iris flower type classification. For a more indepth look at optimizing android for DL4J, please see the Prerequisites and Configuration documentation [here](https://github.com/jrmerwin/Skymind-Android-Documentation/blob/master/Prereqs%20and%20Configuration%20for%20Android.md). This application has a simple UI to take measurements of petal length, petal width, sepal length, and sepal width from the user and returns the probability that the measurements belong to one of three types of Iris (*Iris serosa*, *Iris versicolor*, and *Iris virginica*). A data set includes 150 measurement values (50 for each iris type) and training the model takes anywhere from 5-20 seconds, depending on the device.
+
+Contents
+
+* [Setting the Dependencies](#head_link1)
+* [Setting up the neural network on a background thread](#head_link2)
+* [Preparing the training data set and user input](#head_link3)
+* [Building and Training the Neural Network](#head_link4)
+* [Updating the UI](#head_link5)
+* [Conclusion](#head_link6)
+
+## DL4JIrisClassifierDemo
+![](images/screen.PNG)
+## <a name="head_link1">Setting the Dependencies</a>
+Deeplearning4J applications require several dependencies in the build.gradle file. The Deeplearning library in turn depends on the libraries of ND4J and OpenBLAS, thus these must also be added to the dependencies declaration. Starting with Android Studio 3.0, annotationProcessors need to be defined as well, requiring dependencies for -x86 or -arm processors. 
+```java
+	compile 'com.android.support:appcompat-v7:27.0.2'
+        compile 'com.android.support:design:27.0.2'
+        compile 'org.deeplearning4j:deeplearning4j-nn:0.9.1'
+        compile 'org.nd4j:nd4j-native:0.9.1'
+        compile 'org.nd4j:nd4j-native:0.9.1:android-x86'
+        compile 'org.nd4j:nd4j-native:0.9.1:android-arm'
+        compile 'org.bytedeco.javacpp-presets:systems-platform:1.4'
```

Comment:
Please replace all the `'org.bytedeco.javacpp-presets:systems-platform:1.4'` with `'org.bytedeco:javacpp:1.4'`